env.d/development/common.dist

```diff
@@ -53,3 +53,6 @@ JOANIE_BACKOFFICE_BASE_URL="http://localhost:8072"
 
 # must be reachable for signature email notifications
 DEVELOPER_EMAIL="developer@example.com"
+
+# Security for remote endpoints API
+JOANIE_AUTHORIZED_API_TOKENS = "first item, second item"
```

Comment:
Let's use realistic value. I don't think a token can have spacing chars.
```suggestion
JOANIE_AUTHORIZED_API_TOKENS = "secretTokenForRemoteAPIConsumer"
```

Reply:
Just to clarify my point, here we have a valueList, so for providing mulltiple items  with the default separator `, `, we must use something like `token_1, token_2`.